hvplot/converter.py

```diff
@@ -1962,13 +1984,19 @@ def method_wrapper(ds, x, y):
             layers = _transfer_opts_cur_backend(layers)
             return layers
 
-        import_datashader()
+        ds = import_datashader()
         from holoviews.operation.datashader import datashade, rasterize, dynspread
 
         categorical, agg = self._process_categorical_datashader()
         if agg:
             opts['aggregator'] = agg
-
+        if self.selector:
+            selector = self.selector
+            if isinstance(selector, str):
+                selector = getattr(ds, selector)()
```

Comment:
Maybe have some error handling here, so if people enter the wrong string, it would provide more helpful text. 

Right now, it will just raise a pretty obscure AttributeError.